patroni/ctl.py

```diff
@@ -800,3 +805,41 @@ def flush(cluster_name, member_names, config_file, dcs, force, role, target):
                 check_response(r, member.name, 'flush scheduled restart')
             else:
                 click.echo('No scheduled restart for member {0}'.format(member.name))
+
+
+@ctl.command('disable', help='Disable auto failover')
```

Comment:
To me it makes sense to call it pause, since we use 'pause' all over the code (basically, the only place where we call the operation ``disable`` is in ``patronictl``, every other user-visible message uses ``pause``.

Also, we seem to omit TTL from the implementation, is it intentionally?